<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SIDEAS - Simulasi Decoder ADS-B</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelBackground">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1921</width>
      <height>1081</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="pixmap">
     <pixmap>../assets/lapanXpens.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonSimulation">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>610</y>
      <width>451</width>
      <height>171</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Radley</family>
      <pointsize>35</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #5199DC;
border: no;
border-radius: 20px;
</string>
    </property>
    <property name="text">
     <string>SIMULATION
 ADS-B</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonAbout">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>390</y>
      <width>451</width>
      <height>161</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Radley</family>
      <pointsize>35</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #5199DC;
border: no;
border-radius: 20px;</string>
    </property>
    <property name="text">
     <string>ABOUT US</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonBack">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto Condensed</family>
      <pointsize>21</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">file:///C:/Users/USER/PycharmProjects/ADS-B Simple Decoder/assets/back-arrow.png;
</string>
    </property>
    <property name="text">
     <string>BACK</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
